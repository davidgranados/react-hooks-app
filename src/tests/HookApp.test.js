import React from "react";
import { shallow } from "enzyme";
import HookApp from "../HooksApp";

describe("Tests on HookApp", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
