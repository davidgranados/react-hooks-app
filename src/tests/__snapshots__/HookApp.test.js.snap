// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;

exports[`Tests on HookApp should match snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
